### ISSUE TITLE
Define a process for concurrent development

### ISSUE THREAD

**Issue by asapnet** [Veremes/rTest#7]
FME workspaces and linked custom transformers are not practically mergeable, so multiple people working on it might be a challenge.  It would be nice in the long term if we could define a process and tools to allow us to safely share in rTest's development.  I was thinking of breaking some of its functionality into multiple linked custom transformers with the existing player workspace being the master.  Then creating scripts in my own open source repository which we could use here to manage file permissions and effectively provide a dashboard of who's working on what so we don't accidentally step on each other.  More details to follow.

**Comment by ogayte** [comment]
It is very interesting.
I would be a rare example of co-development of a single FME workspace.

**Comment by asapnet** [comment]
Perhaps you haven't had this come up yet, but we have, and it's becoming more of a common problem than you might realize.  We have nearly 60 different workspaces, but a handful are core to our processes and are touched by many different fix and enhancement efforts.  And if you want me and others to contribute to **rTest**, we will need a process for managing concurrent development anyway.  Again, this is something I can take on describing, then share with you for comments and concerns before I implement it in in my open source repo for my own projects and/or **rTest**.

**Comment by ogayte** [comment]
Yes, feel free to manage the project as you want. Do you need any specific privilege on the project?

**Comment by asapnet** [comment]
Thanks!  Collaborator permissions would be useful.  If Veremes is an organization account, then Admin on the repo would be necessary if I need to set up webhooks and/or service hooks; but if you can do that for me when the time comes, Write permission on the repo would be sufficient.

**Comment by ogayte** [comment]
I have asked to my colleagues Alexandre and Armand to follow the project and give you assistance. They will be more efficient and reactive.

**Comment by asapnet** [comment]
Thanks!  I am working on testing some git locking infrastructure, but will traveling for the rest of June starting Friday, so I may not get anything to your team about the process and tools until July.  But it will come.
